dist/main.js

```diff
@@ -0,0 +1,42 @@
+const renderer = new Renderer()
+const $start = $('#start')
+
+let board
```

Comment:
you aren't using this variable globally, so declare it inside the relevant function